sdk/messaging_servicebus/src/service_bus/mod.rs

```diff
@@ -21,6 +25,7 @@ use url::form_urlencoded::{self, Serializer};
 
 /// Default duration for the SAS token in days — We might want to make this configurable at some point
 const DEFAULT_SAS_DURATION: u64 = 3_600; // seconds = 1 hour
+pub const BROKER_PROPERTIES: HeaderName = HeaderName::from_static("brokerproperties");
```

Comment:
Nit: why is this public? Better to expose less and open it up as needed that expose it prematurely and have to support it forever (appreciating that this isn't supported anyway, but in general).

Reply:
you are write it probably does not have to be